src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/doclets_ja.properties

```diff
@@ -107,8 +107,8 @@ doclet.Since=導入されたバージョン:
 doclet.Throws=例外:
 doclet.Version=バージョン:
 doclet.Factory=ファクトリ:
-doclet.UnknownTag={0}は不明なタグです。
-doclet.UnknownTagLowercase={0}は不明なタグです。大文字と小文字の区別を除いては既知のタグと同じです。
+doclet.UnknownTag=不明なタグ。未登録のカスタム・タグ?
+doclet.UnknownTagWithHint=不明なタグ。入力ミスによる@{0}または未登録のカスタム・タグ?
```

Comment:
Simply appending a question mark may read a bit odd, although it is a straightforward translation from English.